src/typeGuards.ts

```diff
@@ -14,3 +14,9 @@ export const isSqsEvent = (event: unknown): event is SQSEvent => {
         undefined
     )
 }
+
+export const isScheduledEvent = (event: unknown): event is ScheduledEvent => {
+    const e = event as ScheduledEvent
+    // https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents.html
+    return e?.["detail-type"].toLowerCase() === "scheduled event"
```

Comment:
It's probably not necessary, but should we add `?.toLowerCase()`, just to be more safe?

Reply:
yep, can't hurt